index.php

```diff
@@ -41,11 +41,11 @@
 require_once "../../redcap_connect.php";
 require_once "common.php";
 
-$log_file = "/var/log/redcap/autonotify2.log";
+// $log_file = "/var/log/redcap/autonotify2.log";
```

Comment:
Forces fallback to logging in the temp directory. I'm open to logging elsewhere on the server, but we would obviously need to create the log.